### PR TITLE
Hook up tide to redhat-release-coreos

### PR DESCRIPTION
**Pull request by jlebon**

### CODE REVIEW

**Comment by smarterclayton** [comment]
You need to add yourself to plugins.yaml as well (to get triggers)

**Comment by smarterclayton** [comment]
If you want test protection on merge you need at least one prow job defined.  Otherwise tide will merge you if your tests are green, but not be able to rerun those tests.

**Comment by jlebon** [comment]
Thanks! Updated to also add to `plugins.yaml`.
If we don't want to add any tests for now, will tide still go off of the lgtm label?

**Comment by kargakis** [comment]
Yes, as soon as you lgtm, and assuming all commit contexts are green, tide will merge.

File: cluster/ci/config/prow/plugins.yaml [hunk 1]
```diff
@@ -189,6 +189,9 @@ plugins:
   openshift/os:
   - trigger
 
+  openshift/redhat-release-coreos:
+  - trigger
```
Comment (kargakis) [inline cluster/ci/config/prow/plugins.yaml]: If you don't have prow tests, you don't need this.

**Comment by jlebon** [comment]
OK, removed from `plugins.yaml` again.

**Comment by kargakis** [comment]
Just ensure the bots' team has access in the repo.

/lgtm

**Pull request merged**